model_analyzer/monitor/dcgm/dcgm_monitor.py

```diff
@@ -104,16 +104,18 @@ def _collect_records(self):
         for gpu in self._gpus:
             device_id = gpu.device_id()
             metrics = self.group_watcher.values[device_id]
-
             # Find the first key in the metrics dictionary to find the
             # dictionary length
             first_key = list(metrics)[0]
             num_metrics = len(metrics[first_key].values)
             for i in range(num_metrics):
                 for tag in self._tags:
                     dcgm_field = self.MODEL_ANALYZER_TO_DCGM_FIELD[tag]
+                    # DCGM timestamps are in nano seconds, Record's timestamps
```

Comment:
If a comment come after a line of code, there should be an empty line before it. The exception is if there it is the first line in a code block (i.e. the start of a for loop)